### PR TITLE
Add recipe for toolz

### PR DESCRIPTION
**Pull request by jcrist**

### CODE REVIEW

File: recipes/toolz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set version = "0.7.4" %}
+
+package:
+  name: toolz
+  version: {{ version }}
+
+source:
+  fn: toolz-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/r/toolz/toolz-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/toolz/meta.yaml]: `/r/` -> `/t/`

File: recipes/toolz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+about:
+  home: http://toolz.readthedocs.org/
+  license: BSD 3-Clause
+  summary: A functional standard library for Python.
+
+extra:
+  recipe-maintainers:
+    - mrocklin
+    - eriknw
```
Comment (jakirkham) [inline recipes/toolz/meta.yaml]: Happy to be added as a maintainer, @eriknw?

**Comment by jakirkham** [comment]
LGTM

File: recipes/toolz/meta.yaml (hunk 2 above)
Comment (eriknw) [inline recipes/toolz/meta.yaml]: Yes, thank you!

**Comment by jakirkham** [comment]
Thanks @jcrist.

**Comment by jakirkham** [comment]
Just FYI, when you do bump the version for the `toolz` release, please make sure to fork the feedstock, add changes to a branch on your fork, and PR them back to the main feedstock.

**Pull request merged**